src/aws/cis-1.5.0/README.md

```diff
@@ -0,0 +1,60 @@
+# CIS Amazon Web Services Foundations 1.5.0
+
+Policy Pack based on the [AWS Foundations 1.5.0](https://docs.aws.amazon.com/audit-manager/latest/userguide/CIS-1-5.html) benchmark provided by the [Center for Internet Security (CIS)](https://www.cisecurity.org/benchmark/amazon_web_services/)
```

Comment:
Note: This CIS-1-5 page does not exist

Reply:
We should avoid pointing to a page that doesnt exist. I would find another page that has the benchmark defined, maybe on the CIS website

Reply:
@tyler-dunkel looks like the latest version for the aws cis benchmark is 1.4.0.
Here is the page I found on cis website: https://www.cisecurity.org/benchmark/amazon_web_services

Reply:
See Jira ticket for latest, it might not have a link anywhere yet so you can leave out that link (lets not link to somewhere that doesnt exist)

Reply:
@tyler-dunkel link in README updated